docs/src/pages/ui/primitives/passwordfield/react.mdx

```diff
@@ -0,0 +1,345 @@
+```
+
+<Example>
+  <PasswordField label="Password" name="password" />
+</Example>
+
+### Hiding the show password button
+
+Use the `hideShowPassword` prop to hide the show password button.
+
+```jsx
+<PasswordField label="Password" name="password" hideShowPassword={true} />
+```
+
+<PasswordField label="Password" name="password" hideShowPassword={true} />
+
+### Autocomplete - supporting password managers
+
+Use the `autoComplete` prop to tell browser how to populate a password field. Options for `autoComplete` are `current-password` (default) and `new-password`.
+
+#### How to choose an `autoComplete` option
+
+If customers are logging in, you don't need to do set `autoComplete` because `current-password` is already the default.
+If customers are signing up for a new account, set `autoComplete` to `new-password`.
+
+See [MDN for more information on autocomplete](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete).
+
+_Signup form example: `new-password`_
+
+```jsx
+<form>
+  <Flex as="form" direction="column" gap="1rem">
+    <Heading level={3}>Sign Up</Heading>
+    <TextField label="Username" name="username" autoComplete="username" />
+    <PasswordField
+      label="Password"
+      name="password"
+      autoComplete="new-password"
+      descriptiveText="Password must be at least 8 characters"
+    />
+    <Button type="submit">Sign Up</Button>
+  </Flex>
+</form>
+```
+
+<Example>
+  <SignUpFormExample />
+</Example>
+
+_Login form example: `current-password` (default)_
+
+```jsx
+<Flex as="form" direction="column" gap="1rem">
+  <Heading level={3}>Login</Heading>
+  <TextField label="Username" name="username" autoComplete="username" />
+  <PasswordField
+    label="Password"
+    name="password"
+    autoComplete="current-password"
+  />
+  <Button type="submit" onClick={(e) => e.preventDefault()}>
+    Login
+  </Button>
+</Flex>
+```
+
+<Example>
+  <LoginFormExample />
+</Example>
+
+_Change password example: `current-password` and `new-password`_
+
+```jsx
+<Flex as="form" direction="column" gap="1rem">
+  <Heading level={3}>Change Password</Heading>
+  <TextField label="Username" name="username" autoComplete="username" />
+  <PasswordField
+    label="Current password"
+    name="current_password"
+    autoComplete="current-password"
+    descriptiveText="Password must be at least 8 characters"
+  />
+  <PasswordField
+    label="New password"
+    name="new_password"
+    autoComplete="new-password"
+    descriptiveText="Password must be at least 8 characters"
+  />
+  <PasswordField
+    label="Confirm password"
+    name="confirm_password"
+    autoComplete="new-password"
+  />
+  <Button type="submit">Submit</Button>
+</Flex>
+```
+
+<Example>
+  <ChangePasswordFormExample />
+</Example>
+
+### Accessibility / Label behavior
+
+The PasswordField primitive is accessible by default. A matching `label` HTML element is rendered with the `PasswordField` by default,
```

Comment:
The first 2 sentences both say 'by default', maybe we can combine them or something?

Reply:
Default default default, just trying to get my point across 🤣. Will update.